minifi-toolkit/minifi-toolkit-configuration/src/test/java/org/apache/nifi/minifi/toolkit/configuration/dto/ProcessorSchemaTest.java

```diff
@@ -45,6 +45,35 @@
     private final String testKey = "testKey";
     private final String testValue = "testValue";
     private final String testPenalizationPeriod = "55 s";
+    private final String testAnnotationData = ">&lt;criteria&gt;\n" +
```

Comment:
Ah good catch, it should be removed. I mis-copied it from a the xml of a template. Could you just amend the commit to remove that? 

Sorry, I'm in the middle of debugging something else